deploy-manage/upgrade/prepare-to-upgrade.md

```diff
@@ -73,7 +75,36 @@ Optionally create a 9.0.0 deployment and reindex from remote:
 1. Provision an additional deployment running 9.0.0.
 2. To reindex your data into the new {{es}} cluster, use the [reindex documents API](https://www.elastic.co/docs/api/doc/elasticsearch/v8/operation/operation-reindex) and temporarily send new index requests to both clusters.
 3. Verify the new cluster performs as expected, fix any problems, and then permanently swap in the new cluster.
-4. Delete the old deployment. On {ecloud}, you are billed only for the time the new deployment runs in parallel with your old deployment. Usage is billed on an hourly basis.
+4. Delete the old deployment. On {{ecloud}}, you are billed only for the time the new deployment runs in parallel with your old deployment. Usage is billed on an hourly basis.
+
+## Upgrade uni-directional {{ccr}} clusters with followed data streams [upgrade-ccr-data-streams]
+
+When moving to a new major version of {{es}}, you must perform specific actions to ensure that indices — including those that back a data stream — are compatible with the latest Lucene version. With a {{ccr-init}}-enabled cluster, consider whether you want to keep your older data writable or read only to ensure you make changes to the cluster in the correct order. 
+
+:::{note}
+{{ccr-init}}-replicated data streams only allow writing to the most recent backing index as ILM automatically injects an unfollow event after every rollover. Therefore, you can't reindex data streams while {{ccr-init}} is replicating the target backing indices.
```

Comment:
This is slightly misleading as once ILM unfollows, CCR technically isn't following those backing indices anymore and that's the issue as the reindex will not propagate.
```suggestion
{{ccr-init}}-replicated data streams only allow writing to the most recent backing index as ILM automatically injects an unfollow event after every rollover. Therefore, you can't reindex {{ccr-init}} followed data streams as older backing indices are no longer replicated by {{ccr-init}}.
```